app/src/main/java/org/oppia/app/activity/ActivityComponent.kt

```diff
@@ -19,5 +20,6 @@ interface ActivityComponent {
 
   fun getFragmentComponentBuilderProvider(): Provider<FragmentComponent.Builder>
 
+  fun inject(homeActivity: HelpActivity)
```

Comment:
rename this as 
`fun inject(helpActivity: HelpActivity)`

Reply:
resolved